src/firefly/js/visualize/ui/ImageSearchPanelV2.jsx

```diff
@@ -306,12 +306,12 @@ function SelectArchive({groupKey,  imageMasterData, multiSelect}) {
                                      initialState={{
                                              unit: 'arcsec',
                                              labelWidth : 0,
-                                             nullAllowed: false,
+                                             nullAllowed: true,
```

Comment:
I think we need to do more than `nullAllowed`. We probably should have a radio box that has 2 options `full image` or `cutout`